diff --git a/frontend/app/_layout.tsx b/frontend/app/_layout.tsx
index cca853c..d97e3a8 100644
--- a/frontend/app/_layout.tsx
+++ b/frontend/app/_layout.tsx
@@ -56,6 +56,7 @@ export default function RootLayout() {
               <Stack.Screen name="(tabs)" />
               <Stack.Screen name="bayi/[id]" />
               <Stack.Screen name="fatura/[id]" />
+              <Stack.Screen name="satis/[id]" />
             </Stack>
           </View>
         </AuthProvider>
